### PR TITLE
docs: Update references

### PR DESCRIPTION
**Pull request by Techassi**
Part of https://github.com/stackabletech/stackable-cockpit/pull/101

### CODE REVIEW

File: docs/modules/airflow/pages/index.adoc [hunk 1]
```diff
@@ -2,47 +2,73 @@
 :description: The Stackable Operator for Apache Airflow is a Kubernetes operator that can manage Apache Airflow clusters. Learn about its features, resources, dependencies and demos, and see the list of supported Airflow versions.
 :keywords: Stackable Operator, Apache Airflow, Kubernetes, k8s, operator, engineer, big data, metadata, job pipeline, scheduler, workflow, ETL
 
+:k8s-crs: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
```
Comment (fhennig) [inline docs/modules/airflow/pages/index.adoc:5]: why did you add the link like this?

you know you can just use `https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Link Text]` right?

File: tests/test-definition.yaml
Comment (fhennig) [inline tests/test-definition.yaml:1]: why is all this stuff commented out?

File: docs/modules/airflow/pages/index.adoc (hunk 1 above)
Comment (Techassi) [reply docs/modules/airflow/pages/index.adoc:5]: Way too long... That's why I use link references.

File: tests/test-definition.yaml
Comment (Techassi) [reply tests/test-definition.yaml:1]: I was basing my branch off of an old state. I just merged changes from `main` and now that stuff is not commented out anymore.

**Review by fhennig — APPROVED** [review]
